docs/technical-details/reference/components/managing-applications.md

```diff
@@ -1,20 +1,22 @@
-# Managing
+# Managing Applications
```

Comment:
I wonder if this should be just "Applications"? We have "Extensions" and "Plugins" under this "Components" section, so seeing "Applications" could drive home that they are just another type of component.